data/en/test_checkbox_mutually_exclusive.json

```diff
@@ -20,61 +20,101 @@
-                            "value": "No extra",
-                            "q_code": "7"
+                            "parent_answer_id": "mandatory-checkbox-answer",
+                            "mandatory": false,
+                            "id": "other-answer-mandatory",
+                            "label": "Please specify other",
+                            "type": "TextField"
+                        }],
+                        "id": "mandatory-checkbox-question",
+                        "title": "Which extra pizza toppings would you like?",
+                        "type": "General"
+                    }],
+                    "title": "Mandatory other option",
+                    "routing_rules": []
+                }, {
+                    "type": "Question",
+                    "id": "optional-checkbox",
+                    "questions": [{
+                        "answers": [{
+                            "id": "optional-checkbox-answer",
+                            "mandatory": false,
+                            "options": [{
+                                    "label": "Cheddar",
+                                    "value": "Cheddar"
+                                },
+                                {
+                                    "label": "Mozzarella",
+                                    "value": "Mozzarella"
+                                },
+                                {
+                                    "label": "I don't like cheese",
+                                    "value": "No cheese"
+                                }
+                            ],
+                            "type": "MutuallyExclusiveCheckbox"
                         }],
-                        "type": "MutuallyExclusiveCheckbox"
-                    }, {
-                        "parent_answer_id": "mandatory-checkbox-answer",
-                        "mandatory": false,
-                        "id": "other-answer-mandatory",
-                        "label": "Please specify other",
-                        "type": "TextField"
+                        "id": "optional-checkbox-question",
+                        "title": "What is your favourite cheese?",
+                        "type": "General"
                     }],
-                    "id": "mandatory-checkbox-question",
-                    "title": "Which extra pizza toppings would you like?",
-                    "type": "General"
-                }],
-                "title": "Mandatory other option",
-                "routing_rules": []
-            }, {
-                "type": "Summary",
-                "id": "summary"
-            }],
+                    "title": "Optional question"
+                },
+                {
+                    "id": "cheese-interstitial",
+                    "title": "Cheese Interstitial Page",
+                    "description": "You Didn't select an option",
```

Comment:
This description should probably be updated.  I had to look at the schema to figure out what this page was actually testing